en/sensor/airspeed.md

```diff
@@ -0,0 +1,30 @@
+# Airspeed Sensors
+
+Airspeed sensors are *highly recommended* for fixed-wing and VTOL frames.
+They are so important because the autopilot does not have other means to detect stall.
+For fixed-wing flight it is the airspeed that guarantees lift not ground speed!
+
+![Digital airspeed sensor](../../images/digital_airspeed_sensor.jpg)
+
+
+## Hardware Options
+
+Recommended digital airspeed sensors include:
```

Comment:
Are there any more we should mention? I added links that made sense based on the driver names and documentation.

Reply:
This looks good. The MS4525DO is also available here: https://drotek.com/shop/en/airspeed/793-digital-differential-airspeed-sensor-kit-.html?search_query=airspeed&results=6.